workflow/envs/rbt.yaml

```diff
@@ -2,5 +2,4 @@ channels:
   - bioconda
   - conda-forge
 dependencies:
-  - rust-bio-tools =0.20
-  - mkl # TODO without this dependency, we get "rbt: error while loading shared libraries: libmkl_rt.so: cannot open shared object file: No such file or directory" on some systems
+  - rust-bio-tools =0.20.3
```

Comment:
As far as i know this shouldn't have any effect. With `rust-bio-tools =0.20` (no minor version specified) the newest version, i.e. `0.20.3` should be automatically used.

Reply:
changed back

Reply:
Nice! This might save us some _Update rbt version to 0.20.x_ commits.